tests/foreman/ui/test_discoveredhost.py

```diff
@@ -19,10 +19,9 @@
 from fauxfactory import gen_string
 from nailgun import entities
 
+from robottelo import ssh
```

Comment:
The correct way is:
```suggestion
from robottelo.utils import ssh
```
Probably we forgot to delete the `robottelo/ssh.py` module. You could remove that in this PR as well :)

Reply:
Looks like there's a few places it needs to be updated as well. I can do that in a separate PR.